node/src/test/test.ts

```diff
@@ -93,6 +93,29 @@ describe("LanceDB client", function () {
       const con = await lancedb.connect(uri);
       assert.deepEqual(await con.tableNames(), ["vectors"]);
     });
+
+    it("read consistency level", async function () {
+      const uri = await createTestDB();
+      const db1 = await lancedb.connect({ uri });
+      const table1 = await db1.openTable("vectors");
+
+      const db2 = await lancedb.connect({
+        uri,
+        readConsistencyInterval: 0
+      })
+      const table2 = await db2.openTable("vectors");
+
+      await table1.add([
```

Comment:
We should make this test more robust by asserting the state before the change as well:
```suggestion
      assert.equal(await table2.countRows(), 2);
      await table1.add([
```